src/libs/humanizer/humanizerModules.test.ts

```diff
@@ -370,12 +361,7 @@ describe('module tests', () => {
       .map((key: string) => transactions[key])
       .flat()
     accountOp.calls = allCalls
-    const [irCalls, asyncOps] = humanizeCalls(
-      accountOp,
-      humanizerModules,
-      humanizerInfo,
-      standartOptions
-    )
+    const irCalls = humanizeAccountOp(accountOp, standartOptions)
     // irCalls.forEach((c: IrCall, i) => {
```

Comment:
This test is skipped because we do not have the textification functionality anymore. It is not deleted as we will need such test for collision detection for the modules. We should rewrite this in separate PR